src/components/sections/ProductGallery/ProductGallery.tsx

```diff
@@ -31,100 +31,114 @@ function ProductGallery({ title }: Props) {
       return
     }
 
-    // notebook breakpoint = 1280px (See breakpoints on styles/global.scss)
-    setIsMobile(screenWidth < 1280)
+    const notebookBreakpoint =
+      getComputedStyle(document.documentElement).getPropertyValue(
+        '--breakpoint-notebook'
+      ) || '1280'
+
+    setIsMobile(screenWidth < parseInt(notebookBreakpoint, 10))
   }, [screenWidth])
 
   if (!data) {
-    return <div className="temp-data-loading">loading...</div>
+    return <div className="plp-temp-data-loading">loading...</div>
   }
 
   return (
-    <>
-      {/* Filters */}
-      <div>
+    <div className="plp-content-grid / grid-content">
```

Comment:
We need to fix this padding on mobile. My proposal:
```suggestion
    <div className="grid-content-full">
```
And create a child `<div className="plp-content-grid / grid-content">`

![image](https://user-images.githubusercontent.com/11613011/150385710-0afacb0c-bef0-44ae-81f5-fba3f237992a.png)a

Reply:
[Done](https://github.com/vtex-sites/base.store/pull/240/commits/b51f7eea171c6dc7b66495ebbcf3f024db0bcd74)!